packages/compiler/src/parser/index.ts

```diff
@@ -57,7 +57,7 @@ export class Parser {
           code: `unclosed-block`,
           message: `Block was left open`,
         },
-        current.start!,
+        current.start! + 1,
```

Comment:
Magic number. What means this 1 in this context?

Reply:
It's just that the `current` index represents the start of the node, not the position of the first character (which is always 1 character after it). So... not really a magic number ig? I don't think it's super relevant in this case. Thoughts?